lib/organisms/gravity-form/gravity-form.controller.js

```diff
@@ -1,11 +1,169 @@
-angular
-    .module('lnPatterns')
-    .controller('lnOGravityFormController', lnOGravityFormController);
+(function () {
```

Comment:
We don't need to do this because browserify will do it for us when it builds the bundle.